model/workflow_validator.go

```diff
@@ -224,3 +220,77 @@ func validationNotExclusiveParamters(values []bool) bool {
 	}
 	return (hasOne && hasTwo) || (!hasOne && !hasTwo)
 }
+
+type WorkflowErrors []error
+
+func (e WorkflowErrors) Error() string {
+	errors := []string{}
+	for _, err := range []error(e) {
+		errors = append(errors, err.Error())
+	}
+	return strings.Join(errors, "\n")
+}
+
+func WorkflowError(err error) error {
```

Comment:
Hey guys. Also start in that PR the issue https://github.com/serverlessworkflow/sdk-go/issues/125. Can you validate these error messages are ok?